sopel/privileges.py

```diff
@@ -1,146 +1,147 @@
-"""Constants for user privileges in channels.
+"""Constants for user privileges in channels."""
+# Copyright 2023 dgw, technobabbl.es
```

Comment:
The commit message for 12b00b0b424e8fb9896ad48b8368cc1fcb51379b explains it. The file might have already existed, but git said "rewrite sopel/privileges.py (97%)" so it's basically all me. Some of the old documentation is left over, I guess, so that number's perhaps a bit high, but I still think it's justified. I did forget the license part, though, so I'll add that.

Poking around, I saw that there are plenty of other files without copyright notices. We should probably fix that, though analyzing each file's history to decide what year(s) and name(s) go in the header could be annoying. Sounds like a good lazy-patch project for 8.1 to me. 😉